DYDX-Bots/Roles_Monitor/src/agent.spec.ts

```diff
@@ -0,0 +1,348 @@
+import { FindingType, FindingSeverity, Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+import { createAddress, TestTransactionEvent } from "forta-agent-tools/lib/tests";
+import { toUtf8Bytes, keccak256, Interface } from "ethers/lib/utils";
+import { provideHandleTransaction } from "./agent";
+import NetworkManager from "./network";
+import { MODULE_IFACE } from "./utils";
+
+const testSender: string = createAddress("0xad");
+
+// toUtf8Bytes because ethers' keccak256 does not accept UTF-8 strings
```

Comment:
My recommendation here is to use `keccack256` from `forta-agent`, it allows regular strings iirc